src/Internal.AspNetCore.Sdk/build/ApiCheck.targets

```diff
@@ -0,0 +1,30 @@
+﻿<!--
+  WARNING: These targets are intended for building Microsoft's ASP.NET Core repos and are not intended
+  for use outside of Microsoft.
+-->
+<Project>
+  <Target Name="ApiCheck" Condition=" '$(EnableApiCheck)' == 'true' ">
+    <PropertyGroup>
+      <_ApiListingFileSuffix Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">net45.json</_ApiListingFileSuffix>
+      <_ApiListingFileSuffix Condition=" '$(_ApiListingFileSuffix)' == '' ">netcore.json</_ApiListingFileSuffix>
+      <_ApiListingFilePath>$(MSBuildProjectDirectory)\baseline.$(_ApiListingFileSuffix)</_ApiListingFilePath>
+      <_ApiExclusionsFilePath>$(MSBuildProjectDirectory)\exceptions.$(_ApiListingFileSuffix)</_ApiExclusionsFilePath>
+      <_ApiListingFile Condition=" Exists('$(_ApiListingFilePath)') ">$(_ApiListingFilePath)</_ApiListingFile>
+      <_ApiExclusionsFile Condition=" Exists('$(_ApiExclusionsFilePath)') ">$(_ApiExclusionsFilePath)</_ApiExclusionsFile>
+    </PropertyGroup>
+
+    <Message Condition=" '$(_ApiListingFile)' != '' "
+             Importance="normal"
+             Text="Checking for breaking changes in $(MSBuildProjectName), $(TargetFramework)" />
+    <ApiCheckTask Condition=" '$(_ApiListingFile)' != '' "
+                  ApiListingPath="$(_ApiListingFile)"
```

Comment:
This is how VS indents XML by default. But, I agree it won't be easy to maintain this in Notepad and VS Code.

I am _not_ changing existing indentation to match. There's a random mix in this repo.